### ISSUE TITLE
[FR] Enable users to set start week on Monday in the calendar picker

### ISSUE THREAD

**Issue by annieappflowy** [AppFlowy-IO/AppFlowy#2157]
### Description

Currently the calendar picker starts week on Sunday
Add an option under `Date format & time format` for the user to set week on Monday in the calendar picker

<img width="345" alt="image" src="https://user-images.githubusercontent.com/12026239/229110219-bb9c2cd8-12e1-46db-8099-9e3f1a2cf3c0.png">

### Impact

Users who are used to see weeks starting on Monday

**Comment by NePoCz** [comment]
Also noticed that you can have inconsistency among calendar picker and/or even individual calendar pages, each starting week at different day. Would it make sense to merge into one setting, probably in the main settings menu?

**Comment by richardshiue** [comment]
> Also noticed that you can have inconsistency among calendar picker and/or even individual calendar pages, each starting week at different day. Would it make sense to merge into one setting, probably in the main settings menu?

Good idea, I agree! Though, our settings design hasn't really solidified yet. What do you think @annieappflowy?

**Comment by annieappflowy** [comment]
I love @NePoCz 's idea of having a global setting for the calendar picker

**Comment by rileyhawk1417** [comment]
![Image](https://github.com/user-attachments/assets/c5f1c3a0-12c2-498e-b2ac-6d3d9090e428)

In 0.8.1 you can manually change the starting date for a calendar. However having a default global calendar setting/value would help.

**Comment by annieappflowy** [comment]
Supported
